beacon-chain/node/node.go

```diff
@@ -684,6 +684,8 @@ func loadChainConfigFile(chainConfigFileName string) {
 		}
 	}
 	yamlFile = []byte(strings.Join(lines, "\n"))
+	lines = strings.Split(string(yamlFile), "\n")
+	log.Info(lines)
```

Comment:
why are you logging this ?

Reply:
sorry , for debugging

Reply:
Should be good to remove, likely was for debugging